elide-spring/pom.xml

```diff
@@ -41,7 +41,6 @@
     <properties>
         <parent.pom.dir>${project.basedir}/../..</parent.pom.dir>
         <spring.boot.version>3.0.1</spring.boot.version>
-        <groovy.version>3.0.13</groovy.version>
```

Comment:
Unused since spring 2.7.0
```
[INFO] +- io.rest-assured:rest-assured:jar:5.2.1:test
[INFO] |  +- org.apache.groovy:groovy:jar:4.0.6:test (version managed from 4.0.1)
[INFO] |  +- org.apache.groovy:groovy-xml:jar:4.0.6:test (version managed from 4.0.1)
[INFO] |  |  \- (org.apache.groovy:groovy:jar:4.0.6:test - version managed from 4.0.1; omitted for duplicate)
```